data/web/public/index.php

```diff
@@ -1,127 +1,279 @@
+                            </tr>
+                            <tr>
+                                <td>Pass</td>
+                                <td><?= $mysqlConfig['client']['password'] ?></td>
+                            </tr>
+                            </tbody>
+                        </table>
+                    </div>
+                    <div class="col-md-4">
+                        <table class="table">
+                            <thead>
+                            <tr>
+                                <th colspan="2">
+                                    Redis
+                                </th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            <tr>
+                                <td>Server</td>
+                                <td>redismaster</td>
+                            </tr>
+                            <tr>
+                                <td>Port</td>
+                                <td><?= $redisConfig['port'] ?></td>
+                            </tr>
+                            </tbody>
+                        </table>
+                    </div>
+                    <div class="col-md-4">
+                        <table class="table">
+                            <thead>
+                            <tr>
+                                <th colspan="2">
+                                    Varnish
+                                </th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            <tr>
+                                <td>Server list</td>
+                                <td>varnish:6082 </td>
+                            </tr>
+                            <tr>
+                                <td>Secret</td>
+                                <td><?= $varnishSecret ?></td>
+                            </tr>
+                            <tr>
+                                <td>Backend Host</td>
+                                <td>varnish</td>
+                            </tr>
+                            <tr>
+                                <td>Backend Port</td>
+                                <td>8080</td>
+                            </tr>
+                            </tbody>
+                        </table>
+                    </div>
                 </div>
             </div>
         </div>
-    </div>
-</header>
-<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
-<script src="https://netdna.bootstrapcdn.com/bootstrap/3.1.1/js/bootstrap.min.js"></script>
-</body>
-</html>
+    </header>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
+    <script src="https://netdna.bootstrapcdn.com/bootstrap/3.1.1/js/bootstrap.min.js"></script>
+    </body>
+    </html>
+    <?php
+endif;
+?>
```

Comment:
That would be a great idea, in the meantime, you could add the minified versions to the HTML file.